pcsx2/VMManager.cpp

```diff
@@ -2532,7 +2530,7 @@ void VMManager::LogCPUCapabilities()
 	}
 #endif
 
-#if 0
+#if defined(__linux__) || defined(_WIN32)
```

Comment:
The issue isn't that it's bash. The invocation of lspci is the hardest hit because it has to enumerate (and wake up) all pci devices. The original issue was the time it might take to wake up a sleeping gpu (quite possible in multi gpu setups). Even with my dedicated GPU awake, a cold lspci call can take ~1.5 seconds. nvidia-smi and glxgears takes ~ 0.5 seconds each as well.

Maybe we can experiment with dispatching this call to a different thread? We'd have to ensure that we don't dump text into the console while the main thread is using it though.

Reply:
To add on to this, the reason it was disabled and not removed is so we can eventually implement something like "Generate support dump" in the toolbar that dumps all of the host information / settings / logs / etc to a file. In the future our discord bot might be able to do some parsing of this theoretical file and show us custom settings, the hw info, maybe the CPU / GPU scores, etc. Advanced stuff like the RPCS3 bot to make support easier. If the GPU info takes 3 seconds to grab during that, it's fine because this is a user triggered event.